packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -420,8 +421,13 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
 
       if (buyQuote) {
         const { availability, reason } = buyQuote.quote.settlementDetails
-        if (availability === 'UNAVAILABLE') {
-          yield put(actions.components.buySell.setStep({ reason, step: 'PAYMENT_ACCOUNT_ERROR' }))
+        if (availability === 'UNAVAILABLE' || reason === 'REQUIRES_UPDATE') {
```

Comment:
(minor): I would move those stings in constants/enum

Reply:
👍🏻 I'll change these in a follow up PR i'm working on.